### PR TITLE
string/trimRight

### PR DESCRIPTION
**Pull request by bookercodes**
Partial implementation of #187.

**Todo**
- [x] Add `chars` argument where `chars`  are custom chars to trim.
- [x] Adhere to linter rules.
- [x] Add docs

### CODE REVIEW

**Comment by bookercodes** [comment]
Alright, @EvanHahn, I think this is good now. 

Please let me know what you think.

Thanks.

File: string/trimRight.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+@returns {string} The given string without trailing chars.
+@example
+var trimRight = require('lildash/string/trimRight');
+trimRight('foo  ');  // => 'foo'
+trimRight('_foo_', '_');  // => '_foo'
+trimRight('!--foo--!', '!-');  // => '!--foo'
+*/
+module.exports = function (str, chars) {
+  chars = chars || ' ';
+  var pattern = new RegExp('([' + chars + ']*$)', 'g');
```
Comment (EvanHahn) [inline string/trimRight.js]: Could we have this function take a regular expression instead of a string? And could it default to `/\s/g`?

**Comment by EvanHahn** [comment]
Could we add tests for trailing tab characters (`\t`)?

File: string/trimRight.js (hunk 1 above)
Comment (bookercodes) [inline string/trimRight.js]: Yeah, I can do that. No problem.

I did it this way because that is how _lodash_ does it.

**Comment by bookercodes** [comment]
@EvanHahn Could you please elaborate? Do you mean that the function should trip trailing tabs **by default**?

**Comment by EvanHahn** [comment]
@alexbooker Yes—it should strip trailing tabs by default.

File: string/trimRight.js (hunk 1 above)
Comment (bookercodes) [inline string/trimRight.js]: Are you sure you want to deviate from the familiar _lodash_ API? 

If yes, I assume you mean for the default pattern to ultimately\* equal _`/\s*$/g`_?

Should I take the given `pattern` and suffix it with _`*$`_ or should the user specify the entire pattern?

(If you can write a couple of test cases that should pass, then that would likely make things very clear for me. Thank you.)

Comment (EvanHahn) [inline string/trimRight.js]: I'd actually feel comfortable with no second argument at all to start. I'm okay deviating from Lodash in several spots.

**Comment by bookercodes** [comment]
@EvanHahn Done and done!

File: string/trimRight.js (hunk 1 above)
Comment (bookercodes) [inline string/trimRight.js]: @EvanHahn That makes me feel comfortable, too. In many cases, it is _much_ simpler to implement functions that have an arity of 1.

**Comment by EvanHahn** [comment]
Perfect! Thank you.

**Pull request merged**